supabase/migrations/20241118122106_add-updated-dat-on-topics.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE topics
+    ADD COLUMN updated_at timestamptz;
+
+UPDATE
+    topics
+SET
+    updated_at =(
+        SELECT
+            CASE WHEN updated_at IS NOT NULL THEN
+                updated_at
+            ELSE
+                created_at
+            END
+        FROM
+            messages
+        WHERE
+            topics.last_message_id = messages.id);
+
+ALTER TABLE attachments
+    ADD COLUMN width int,
+    ADD COLUMN height int;
```

Comment:
You could add checks/constraints to ensure they'll always be above or equal to `0`